apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -16,15 +16,13 @@
  * under the License.
  */
 
-import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
-import { addAlert } from "@wso2is/core/store";
+import { TestableComponentInterface } from "@wso2is/core/models";
```

Comment:
Let's use `IdentifiableComponentInterface` as `TestableComponentInterface` is now deprecated.

Reply:
I need to conditionally disable each checkbox based on the value of the other one (when top one is unchecked bottom one should be disabled and when bottom one is checked top one should be disabled). For this i need to maintain the state of each checkbox